toonz/sources/CMakeLists.txt

```diff
@@ -198,13 +219,65 @@ else()
 
     find_package(LZO REQUIRED)
     message("LZO:" ${LZO_INCLUDE_DIR})
+
+elseif(UNIX)
+    find_library(GL_LIB GL)
+    find_library(GLU_LIB GLU)
+    find_library(GLUT_LIB glut)
+    find_library(Z_LIB z)
+
+    find_package(TIFF REQUIRED)
+    set(TIFF_LIB ${TIFF_LIBRARY})
+    # the libraries have not .pc nor preferred Find*.cmake. use custom modules.
+    # find_package(PNG REQUIRED)
+    set(PNG_LIB ${SDKROOT}/libpng-1.6.21/.libs/libpng16.so)
```

Comment:
If anything, there should be a normal option for it IMO. Autodetecting stuff like that and building a software differently based on the result is bad for packaging that software reproducibly (i.e. the packaging process will result in different packages depending on whether libpng is installed on the build system or not. So if some package maintainer is unaware of the autodetection he might create a package without the libpng dependency and someone with system libpng available will end up with a package with incomplete dependency information).